Neos.Neos/Classes/Domain/Service/WorkspaceService.php

```diff
@@ -122,44 +132,64 @@ public function createRootWorkspace(ContentRepositoryId $contentRepositoryId, Wo
                 ContentStreamId::create()
             )
         );
-        $this->metadataAndRoleRepository->addWorkspaceMetadata($contentRepositoryId, $workspaceName, $title, $description, WorkspaceClassification::ROOT, null);
-    }
-
-    /**
-     * Create the "live" root workspace with the default role assignment (users with the role "Neos.Neos:LivePublisher" are collaborators)
-     */
-    public function createLiveWorkspaceIfMissing(ContentRepositoryId $contentRepositoryId): void
-    {
-        $contentRepository = $this->contentRepositoryRegistry->get($contentRepositoryId);
-        $workspaceName = WorkspaceName::forLive();
-        $liveWorkspace = $contentRepository->findWorkspaceByName($workspaceName);
-        if ($liveWorkspace !== null) {
-            // live workspace already exists
-            return;
+        $this->metadataAndRoleRepository->addWorkspaceMetadata($contentRepositoryId, $workspaceName, $title, $description, WorkspaceClassification::ROOT, ownerUserId: null);
```

Comment:
Named argument is not needed here.
```suggestion
        $this->metadataAndRoleRepository->addWorkspaceMetadata($contentRepositoryId, $workspaceName, $title, $description, WorkspaceClassification::ROOT, null);
```

Reply:
but i like it and its more precise what we are doing here ^^ its internal either way:)

Reply:
And I still don't like it. It falsely implies that you skipped a parameter - even if it is internal 😝 . I'm not blocking this, but maybe something to discuss in general (code style wise).

Reply:
Ah, that was the part you mentioned yesterday :)
My personal take (and I agree, we should update our CGL to the modern language features at some point): I totally agree to the change because it clarifies the `null` (while the other parameters were self-explanatory already).
I tend to use named arguments for these cases more and more – not to skip optional arguments (and even if so: for the reader it wouldn't really matter) but to add context